src/Fantomas/CodePrinter.fs

```diff
@@ -4953,6 +4953,13 @@ and genMemberDefn astContext node =
                 | None -> sprintf "abstract %s" s
             |> fun s -> !- s ctx
 
+        let hasGenerics =
```

Comment:
Could you try and avoid having a PR that does two unrelated things?
I'm ok with this change, but it seems unrelated to changing the default settings.

Reply:
It is the same fix as https://github.com/fsprojects/fantomas/pull/1935, right?